app.js

```diff
@@ -26,6 +29,7 @@ app.use(express.json());
 // Unauthenticated routes:
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDoc));
 app.use('/api/users', usersRouter);
+app.use('/api/water', waterRoute);
```

Comment:
keep it the way it is. don't change the route or the name of the const.
Move this line of code to be after:

// Apply auth middleware
app.use('/api', validateAuth);

so that your route has the req.user object